source/turtles.rst

```diff
@@ -165,25 +169,27 @@ Code in files
 
 Most code is written and executed from a file. 
 
-Using SublimeText to create a file named `my_turtle_file.py` with this code:: 
+Use SublimeText to create a file named `my_turtle_file.py` in the documents folder with this code:: 
 
     from turtle import Turtle, exitonclick
 
     tess = Turtle()
     tess.shape("turtle")
     tess.forward(100)
 
-    exitonclick()  # Why this? Experiment by commenting it out.
+    exitonclick()
```

Comment:
Nevermind! https://github.com/gu-lseg/intro-to-programming/pull/16/files#diff-97dc12f37006e6bb72caf4e1836744ecR205